pkg/storage/bench_test.go

```diff
@@ -63,6 +63,8 @@ func BenchmarkMVCCGarbageCollect(b *testing.B) {
 		{2, []int{1}},
 		{1024, []int{1, 16, 32, 512, 1015, 1023}},
 	}
+	numRangeTombstones := []int{0, 1}
```

Comment:
Would be nice to have a variant with 100 random range tombstones too, like the other benchmarks, but this is fine for now.

Reply:
That requires reverting key by key to ts by ts insertion. I did this change when experimenting with batch performance, but I'd rather add it separately.

Reply:
Created #87417